src/main/core/work/event.rs

```diff
@@ -8,51 +9,53 @@ use super::task::TaskRef;
 #[derive(Debug)]
 pub struct Event {
     magic: Magic<Self>,
-    task: TaskRef,
     time: EmulatedTime,
-    src_host_id: HostId,
-    dst_host_id: HostId,
-    src_host_event_id: u64,
+    data: EventData,
     _counter: ObjectCounter,
 }
 
 impl Event {
-    pub fn new(task: TaskRef, time: EmulatedTime, src_host: &Host, dst_host_id: HostId) -> Self {
+    /// A new packet event.
```

Comment:
Expand the comment a bit to explain what a packet event is.

Reply:
Added.

> A new packet event, which is an event for packets arriving from the Internet. Packet events do not include packets on localhost.